### PR TITLE
Ft retrieve comments

### PR DESCRIPTION
**Pull request by sebastien15**
#### What does this PR do?

> this PR retrieve all comments from the database and display the results

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `ft-retrieve-comments`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/allComments in post man make sure it is a GET method

retreive comment

### CODE REVIEW

**Review by habinezadalvan — COMMENTED** [review]
Rebase before merging

File: server/routes/commentRoutes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+import commentController from "../controllers/commentController";
+
+
+router.get('/allComments', commentController.allComments);
+// router.post('/addComment', commentController.saveComment);
```
Comment (habinezadalvan) [inline server/routes/commentRoutes.js]: Could you please remove the comment codes

Comment (sebastien15) [reply server/routes/commentRoutes.js]: > Could you please remove the comment codes

>Removed all commented lines

Comment (sebastien15) [reply server/routes/commentRoutes.js]: that file was outdated and i resolved the conflicts remained

**Pull request closed without merging**